recipe/meta.yaml

```diff
@@ -8,14 +8,17 @@ source:
   sha256: 1524b840e42cf3b909e8f8df67c1724012c7dc7f9d076d4feef2d3eff031e8a0
   patches:
       - omit-local-site-packages.patch  # [osx]
+      - osx64-dist.patch                # [osx and x86_64]
```

Comment:
I agree, for use in `conda-forge` the `x86_64` tag is not needed but adding it makes the recipe match better with the one in `anaconda-recipes`.